pkg/apis/kops/channel.go

```diff
@@ -36,7 +35,7 @@ const (
 )
 
 type Channel struct {
-	v1.TypeMeta `json:",inline"`
+	metav1.TypeMeta `json:",inline"`
 	ObjectMeta  metav1.ObjectMeta `json:"metadata,omitempty"`
```

Comment:
This is the reason `verify-gofmt` fails, can you update?
```suggestion
	ObjectMeta      metav1.ObjectMeta `json:"metadata,omitempty"`
```

Reply:
You are great.

Reply:
Np. 5 spaces to align with `metav1.TypeMeta`.

Reply:
OK, tests are passing now. Please squash the commits and force push.

Reply:
OK.